BUILDING.md

```diff
@@ -40,9 +47,7 @@ To fetch the required packages, using a console type:
 
 ```
 cd <sdk-path>
-tools\bin\sdkmanager "platforms;android-21"
-tools\bin\sdkmanager "platforms;android-27"
-tools\bin\sdkmanager "build-tools;26.0.1
+tools\bin\sdkmanager platforms;android-21 build-tools;26.0.1
```

Comment:
Have you tested without android 27?

Reply:
Yes. I don't have android 27 installed, and the macOS kokoro build doesn't install it either.